### PR TITLE
Adds DMAPI for use with TGS4

### PR DESCRIPTION
**Pull request by AffectedArc07**
**What does this PR do:**
Adds the TGS-DMAPI into our code. This allows our code to be controlled with TGS4.

Image of validation passing:
image

**Changelog:**
No CL as this is entirely backend

### CODE REVIEW

**Comment by AffectedArc07** [comment]
image
Why the fuck does this happen to my DME
***EVERY FUCKING TIME***

**Comment by KasparoVy** [comment]
Far out. I'd love to know (preferrably in the PR description) what this accomplishes, or perhaps have a couple links to the OG PRs thrown in for reference

**Comment by AffectedArc07** [comment]
@KasparoVy 
TG has this thing called TGS4, The TG Server hosting suite. https://github.com/tgstation/tgstation-server
In essence, its a full management panel for BYOND servers, with features such as saved revisions, static configs, built in bots, etc etc.

The reason this was added is so that our code will be compatible with TGS4, because as it currently stands, we are not.

**Comment by AffectedArc07** [comment]
image
I literally did not modify a single DMI
Everything hates me today

**Comment by KasparoVy** [comment]
Brilliant, thanks for the rundown! I appreciate it.

File: paradise.dme [hunk 1]
```diff
@@ -1734,9 +1732,9 @@
 #include "code\modules\mob\living\carbon\human\inventory.dm"
 #include "code\modules\mob\living\carbon\human\life.dm"
 #include "code\modules\mob\living\carbon\human\login.dm"
-#include "code\modules\mob\living\carbon\human\logout.dm"
 #include "code\modules\mob\living\carbon\human\npcs.dm"
 #include "code\modules\mob\living\carbon\human\say.dm"
+#include "code\modules\mob\living\carbon\human\shock.dm"
```
Comment (Fox-McCloud) [inline paradise.dme]: update your darn branch before making a PR.

**Pull request closed without merging**